roads.mss

```diff
@@ -2965,29 +2965,29 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
 .directions::directions {
   [zoom >= 16] {
     // intentionally omitting highway_platform, highway_construction
-    [feature = 'highway_motorway'],
-    [feature = 'highway_motorway_link'],
-    [feature = 'highway_trunk'],
-    [feature = 'highway_trunk_link'],
-    [feature = 'highway_primary'],
-    [feature = 'highway_primary_link'],
-    [feature = 'highway_secondary'],
-    [feature = 'highway_secondary_link'],
-    [feature = 'highway_tertiary'],
-    [feature = 'highway_tertiary_link'],
-    [feature = 'highway_residential'],
-    [feature = 'highway_unclassified'],
-    [feature = 'highway_living_street'],
-    [feature = 'highway_road'],
-    [feature = 'highway_service'],
-    [feature = 'highway_pedestrian'],
-    [feature = 'highway_raceway'],
-    [feature = 'highway_cycleway'],
-    [feature = 'highway_footway'],
-    [feature = 'highway_path'],
-    [feature = 'highway_steps'],
-    [feature = 'highway_track'],
-    [feature = 'highway_bridleway'] {
+    [highway = 'motorway'],
+    [highway = 'motorway_link'],
+    [highway = 'trunk'],
+    [highway = 'trunk_link'],
+    [highway = 'primary'],
+    [highway = 'primary_link'],
+    [highway = 'secondary'],
+    [highway = 'secondary_link'],
+    [highway = 'tertiary'],
+    [highway = 'tertiary_link'],
+    [highway = 'residential'],
+    [highway = 'unclassified'],
+    [highway = 'living_street'],
+    [highway = 'road'],
+    [highway = 'service'],
+    [highway = 'pedestrian'],
+    [highway = 'raceway'],
+    [highway = 'cycleway'],
+    [highway = 'footway'],
+    [highway = 'path'],
+    [highway = 'steps'],
+    [highway = 'track'],
+    [highway = 'bridleway'] {
```

Comment:
Is there a specific reason why you changed all these filters? The `feature = 'highway_*'` was put in deliberately to avoid combinatorial explosion.